ReactTask/src/redux/patient/patientAction.ts

```diff
@@ -14,4 +14,25 @@ const createPatientError = createAction(
   })
 );
 
-export { createPatientRequest, createPatientSuccess, createPatientError };
+const getResolutionsPatientRequest = createAction('doctor/getPatientRequest');
+const getResolutionsPatientSuccess = createAction(
+  'doctor/getPatientSuccess',
```

Comment:
doctor/getPatientSuccessbut action for patient. It would be better to name it patient/getPatientSuccess. Don't you think so?

Reply:
Fixed